### ISSUE TITLE
Calendar app not syncing and wiping out entries in Google calendar

### ISSUE THREAD

**Issue by wagafo** [ubports/ubuntu-touch#870]
- Device: Nexus 5
- Channel: 16.04/rc
- Build: 2018-W35

### Steps to reproduce

1) Clean install 16.04/rc 
2) Enter Google calendar online account in Calendar app
3) Sync with your Google calendar

### Expected behavior

Calendar app and Google calendar should sync.

### Actual behavior

1) No entry of the Google calendar shows up in the Calendar app.
2) After some attempts to sync, all entries with future dates in Google calendar were wiped out

**Comment by hummlbach** [comment]
This has been reported by @YougoChats and by @matv1 too. I just tried to figure out, when we've spoiled that, but cleaning all the evolution and calendar dot directories I know of and going back to the last version that was working for me - developer image 148 from 27th of august - didn't fix it.

**Comment by hummlbach** [comment]
@wagafo oh sorry, I meant they reported it in a telegram group, not as issue here. So your issue is just fine and I only wanted to confirm it.

**Comment by wagafo** [comment]
@hummlbach OK, I asked around the forum and the telegram super group, but nobody reported back as having this issue.

**Comment by matv1** [comment]
I did have this is issue and still seeing the aftermath of it.
So what happenend to me is exactly as described in the bug report, except it was a Nextcloud calendar.

After getting my calendar wiped because of this bug, I thankfully was able to do a restore form a backup file. Also I removed the Nectcloud account from online accounts. 
So everything was back to normal. Then I wanted to re-add my calendar again. So i first re-added my Nextcloud account. That seemed to go well. The account is listed. Then going to the calendar app I want to re-add my calendar there. I hit 'enable calendars'. Then add the account credentials. 
But my calendar isnt appearing . Neither are my events.
However, what is really weird: Calendar items ARE apearing in the pull down calendar ()time & date) from system settings.
So, at some level, the calendar credentials are stored and accesible.
